README.md

```diff
@@ -3,5 +3,17 @@ Server side for the pipsplace site.
 
 Project utilizes a .env file with the following vars:
 
-+ SECRET_KEY: Django project secret key. New keys can be generated by following instructions [here](https://www.educative.io/answers/how-to-generate-a-django-secretkey).
-+ DEBUG: Determines whether Django server launches in debug mode. Options are True or False.
\ No newline at end of file
++ `SECRET_KEY`: Django project secret key. New keys can be generated by following instructions [here](https://www.educative.io/answers/how-to-generate-a-django-secretkey).
++ `DEBUG`: Determines whether Django server launches in debug mode. Options are True or False.
++ `POSTGRES_DB`: name of your database (will be used to name db if none currently exists)
++ `POSTGRES_PASSWORD`: used in conjunction with POSTGRES_USER to authorize connection
++ `POSTGRES_USER`: name of user to login as to db (if db does not currently exist the created superuser will use this name and POSTGRES_PASSWORD)
```

Comment:
Excellent.  Prefixing config with `POSTGRES_` is great to differentiate things that might otherwise be ambigious.